src/lib/ext-monitor.js

```diff
@@ -0,0 +1,91 @@
+export default class ExtensionMonitor {
+  logs = [];
+  extensionMapList = new Map([]);
```

Comment:
It would be good to add an inline comment right above this property to explicitly mention what is the type signature of this Map and what it is used for (looking to the rest of the module this seems to be a `Map<string, Function>` where the `string` key is an extension id and the `Function` value is a listener.

Reply:
This is done